src/content/hi/fundamentals/getting-started/principles/index.markdown

```diff
@@ -0,0 +1,25 @@
+---
+नक़्शा: साझ/संकुचित पृष्ठों की सूची
```

Comment:
The YAML Front Matter attribute should not be translated. The should look something to the effect of:

```
layout: shared/narrow-pages-list
title: "साइट डिजाइन के सिद्धांतों"
description: "गूगल और AnswerLab की जांच उपयोगकर्ताओं को मोबाइल साइटों की एक विविध सेट के साथ बातचीत कैसे एक अध्ययन किया। लक्ष्य सवाल, 'क्या एक अच्छा मोबाइल साइट बनाता जवाब देने के लिए किया गया था?'"
published_on: 2014-08-08
updated_on: 2015-09-17
order: 5
translation_priority: 0
authors:
  - petelepage
```

Check out https://developers.google.com/web/resources/translations/ for the full details on how to translate content.